example-environment.lua

```diff
@@ -64,9 +64,19 @@ local function get_cube_handler(dev)
       U.DEBUG("env", "locked new dev, level: %s, colors: %s", sets.level or "falsy", U.dump(sets.colors))
     elseif sets.dev and event == "roll" then
       if sets.what == "level" then
-        sets.what = sets.colors.capabilities[4] and "ctemp" or sets.colors.capabilities[1] and "eh" or sets.colors.capabilities[0] and "h" or "level"
+        sets.what = sets.colors.capabilities[4] and
```

Comment:
I'd rather have the and-plus-value stuff on a single line, like
A = flag and value
       or otherflag and value
       or otherflag and value
       or fallback

Reply:
Done here and in condition above.